packages/libcap.rb

```diff
@@ -24,22 +16,25 @@ class Libcap < Package
   def self.patch
     # add includes option to make it work with gperf-3.1
     system 'sed -e "/gperf --/s/gperf --/gperf --includes --/" -e "/gperf --/s/cap_lookup_name(/cap_dummy(/" -i libcap/Makefile'
-    # change the path to ld
-    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/ld,g' Make.Rules"
+
+    # change the path to mold
+    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/mold,g' Make.Rules"
+
     # change prefix
     system "sed -i 's,prefix=/usr,prefix=#{CREW_PREFIX},g' Make.Rules"
```

Comment:
Can't we use environment variable overrides for this?

Reply:
I am not sure about it...

Reply:
All good, I'll tidy up the build section in the next libcap update then.